### PR TITLE
 Add announcement for week 49

### PR DESCRIPTION
**Pull request by ubergesundheit**
## Managed Apps

- [x] Giant Swarm Catalog Apps [https://docs.giantswarm.io/changes/managed-apps/](https://docs.giantswarm.io/changes/managed-apps/) ping @giantswarm/team-halo
- [x] Playground Apps [https://docs.giantswarm.io/changes/playground-apps/](https://docs.giantswarm.io/changes/playground-apps/) ping @giantswarm/team-halo
	- [x] Are there any new or upgraded Playground Apps not yet included in the above link?

## Web UI

- [x] Check [https://docs.giantswarm.io/changes/web-ui/](https://docs.giantswarm.io/changes/web-ui/) ping @giantswarm/sig-ux 

*Just talk about the changes since the last announcement. No need to refer to versions as customers are always on the latest anyway.*

## Documentation

- [x] Check [https://docs.giantswarm.io/changes/documentation/](https://docs.giantswarm.io/changes/documentation/) ping @giantswarm/sig-docs 

### Control Plane Kubernetes API and kubectl gs

- [ ] Check [https://docs.giantswarm.io/changes/control-plane-kubernetes-api/](https://docs.giantswarm.io/changes/control-plane-kubernetes-api/) ping @snizhana-dynnyk 
- [ ] Check [https://docs.giantswarm.io/changes/kubectl-gs/](https://docs.giantswarm.io/changes/kubectl-gs/) ping @snizhana-dynnyk 

## Reminders

- Link to docs.giantswarm.io, not the changelogs in repos

### CODE REVIEW

File: announcements/2020-Week-49/announcement.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# :zap: This Week in Provider-Independent Announcements (December 04, 2020):zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed Apps
+
+1. [Kong v1.1.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#110---2020-12-02) provides kong version 2.2.
+1. [cert-exporter v1.4.0](https://github.com/giantswarm/cert-exporter/blob/master/CHANGELOG.md#140---2020-12-02) adds a new metric which tracks expiry of certificates in Kubernetes secrets.
```
Comment (cokiengchiara) [inline announcements/2020-Week-49/announcement.md:8]: @ubergesundheit @glitchcrab cert-exporter is not an optional app and would therefore be included in a tenant cluster release, right? If so, we should not include here 😀

**Comment by marians** [comment]
I'll add stuff for docs and UIs

**Comment by marians** [comment]
I'm done

File: announcements/2020-Week-49/announcement.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# :zap: This Week in Provider-Independent Announcements (December 04, 2020):zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed Apps
+
+- [Kong v1.1.0](https://docs.giantswarm.io/changes/managed-apps/kong-app/v1.1.0/) provides kong version 2.2.
```
Comment (cokiengchiara) [inline announcements/2020-Week-49/announcement.md:7]:
```suggestion
- [Kong v1.1.0](https://docs.giantswarm.io/changes/managed-apps/kong-app/v1.1.0/) upgrades to Kong upstream chart v1.12.0 containing Kong app v2.2.
```

cc @giantswarm/app-squad-kong

File: announcements/2020-Week-49/announcement.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- [Vertical Pod Autoscaler](https://github.com/giantswarm/vertical-pod-autoscaler-app/blob/master/CHANGELOG.md#100---2020-11-27) is now available as a Playground Catalog app. It automatically adjusts the amount of CPU and memory requested by pods running in the Kubernetes Cluster.
```
Comment (cokiengchiara) [inline announcements/2020-Week-49/announcement.md:8]: @TheoBrigitte

File: announcements/2020-Week-49/announcement.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# :zap: This Week in Provider-Independent Announcements (December 04, 2020):zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed Apps
+
+- [Kong v1.1.0](https://docs.giantswarm.io/changes/managed-apps/kong-app/v1.1.0/) upgrades to Kong upstream chart v1.12.0 containing Kong app v2.2.
```
Comment (cokiengchiara) [inline announcements/2020-Week-49/announcement.md:7]:
```suggestion
- [Kong v1.1.0](https://docs.giantswarm.io/changes/managed-apps/kong-app/v1.1.0/) upgrades to Kong app v2.2.
```

File: announcements/2020-Week-49/announcement.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# :zap: This Week in Provider-Independent Announcements (December 04, 2020):zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed Apps
+
+- [Kong v1.1.0](https://docs.giantswarm.io/changes/managed-apps/kong-app/v1.1.0/) upgrades to Kong app v2.2.
+- [Vertical Pod Autoscaler](https://github.com/giantswarm/vertical-pod-autoscaler-app/blob/master/CHANGELOG.md#100---2020-11-27) is now available as a Playground Catalog app. It automatically adjusts the amount of CPU and memory requested by pods running in the Kubernetes Cluster.
```
Comment (cokiengchiara) [inline announcements/2020-Week-49/announcement.md:8]:
```suggestion
- Vertical Pod Autoscaler is now available as a Playground Catalog app. It automatically adjusts the amount of CPU and memory requested by pods running in the Kubernetes Cluster. 
```

Remove link. It shows preview of changelog in Slack and is distracting. And nothing to see in the repo anyway.

**Comment by cokiengchiara** [comment]
thanks @marians ! Please approve when you're fine with it 👍

**Review by cokiengchiara — APPROVED** [review]

**Comment by cokiengchiara** [comment]
@ubergesundheit want to merge this? ready to announce

**Pull request merged**